src/plugins/drawing-tool/components/drawing-layer.tsx

```diff
@@ -137,20 +135,6 @@ export class DrawingLayerView extends React.Component<DrawingLayerViewProps, Dra
     this.setCurrentTool(tool);
   }
 
-  public addListeners() {
-    window.addEventListener("keyup", (e) => {
-      if (!this.props.readOnly) {
-        switch (e.key) {
-          case "Backspace":
-          case "Delete":
-          case "Del":             // IE 9 and maybe Edge
-            this.handleDelete();
```

Comment:
Is the drawing-layer's handle delete still used after this listener goes away?

Reply:
Nope. Good catch, I deleted it.